plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueConverter.java

```diff
@@ -176,7 +181,12 @@ else if (isIcebergTable(tableParameters)) {
         }
         else {
             boolean isCsv = sd.serdeInfo() != null && HiveStorageFormat.CSV.getSerde().equals(sd.serdeInfo().serializationLibrary());
-            dataColumns = fromGlueColumns(sd.columns(), ColumnType.DATA, isCsv);
+            if (isDeltaLakeTable(tableParameters) && ignoreDeltaLakeColumns) {
```

Comment:
We should always ignore columns for 

```
if (isIcebergTable(table) || isDeltaLakeTable(table))
```

I'm not sure whether `ignoreDeltaLakeColumns` check is actually relevant.